service/server.py

```diff
@@ -88,7 +88,7 @@ def get_title(title_ref):
 
 @app.route('/title_search_postcode/<postcode>', methods=['GET'])
 def get_properties(postcode):
-    postcode.replace("_", " ")
+    postcode.replace("_", "")
```

Comment:
On a second thought, does this do anything? The result isn't assigned to any variable.